docs/index.md

```diff
@@ -47,6 +47,8 @@
   - [xAddressToClassicAddress](#xaddresstoclassicaddress)
   - [renameCounterpartyToIssuer](#renamecounterpartytoissuer)
   - [formatBidsAndAsks](#formatbidsandasks)
+- [XRPL Faucets](#xrpl-faucets)
```

Comment:
I don't think we can edit docs/index.md directly. It is a generated file (using the docgen script): https://github.com/ripple/ripple-lib#generating-documentation

Reply:
Thanks I removed the modification I made before.